client/my-sites/hosting/site-backup-card/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { localize } from 'i18n-calypso';
+import { Card } from '@automattic/components';
+import { connect } from 'react-redux';
+import { flowRight as compose } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import isRequestingRewindBackups from 'state/selectors/is-requesting-rewind-backups';
+import getLastGoodRewindBackup from 'state/selectors/get-last-good-rewind-backup';
+import { requestRewindBackups } from 'state/rewind/backups/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getSiteGmtOffset from 'state/selectors/get-site-gmt-offset';
+import getSiteTimezoneValue from 'state/selectors/get-site-timezone-value';
+import { applySiteOffset } from 'lib/site/timezone';
+import { withLocalizedMoment } from 'components/localized-moment';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import classNames from 'classnames';
+
+const SiteBackupCard = ( {
+	disabled,
+	gmtOffset,
+	isRequestingBackups,
+	lastGoodBackup,
+	moment,
+	requestBackups,
+	siteId,
+	timezone,
+	translate,
+} ) => {
+	useEffect( () => {
+		if ( ! disabled && lastGoodBackup === null && ! isRequestingBackups ) {
+			requestBackups( siteId );
+		}
+	}, [ disabled, lastGoodBackup, isRequestingBackups, siteId ] );
+
+	const lastGoodBackupTime = lastGoodBackup
+		? applySiteOffset( moment.utc( lastGoodBackup.last_updated, 'YYYY-MM-DD hh:mma' ), {
+				timezone,
+				gmtOffset,
+		  } )
+		: null;
+
+	return (
+		<Card className={ classNames( 'site-backup-card', { [ 'is-disabled' ]: disabled } ) }>
+			<CardHeading>{ translate( 'Site backup' ) }</CardHeading>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Site Backups' )` *ES Score: 7*